catalog/imaging/catalog-video-transformer/src/main/java/com/connexta/transformer/video/GeometryUtilityImpl.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) Connexta, LLC
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformer.video;
+
+import java.util.Optional;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.vividsolutions.jts.geom.Geometry;
+import com.vividsolutions.jts.io.ParseException;
+import com.vividsolutions.jts.io.WKTReader;
+import com.vividsolutions.jts.io.WKTWriter;
+
+import ddf.catalog.data.Attribute;
+
+public class GeometryUtilityImpl implements GeometryUtility {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(GeometryUtilityImpl.class);
+
+    public GeometryUtilityImpl() {
+
+    }
+
+    @Override
+    public Optional<String> createUnionOfGeometryAttribute(WKTReader wktReader, WKTWriter wktWriter,
```

Comment:
Have you considered making this a static utility method? I'm not sure what is gained by separating this into an interface + implementation.